atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -45,9 +45,9 @@
                 }
                 failureCount++;
                 if (shouldStopRetrying(failureCount)) {
-                    String msg = "Failing after " + failureCount + " tries";
-                    log.warn(msg, e);
-                    throw Throwables.rewrap(msg, e);
+                    String msg = "Failing after {} tries";
+                    log.warn(msg, failureCount, e);
+                    throw Throwables.rewrap(msg + failureCount, e);
```

Comment:
This is going to appear as `"Failing after {} tries 5"`.

Since String.format and the logger use different placeholders it is difficult to reuse the code between these 2 lines.  Either construct the full String once, or create the message separately for each line.  The code is minimal and on consecutive lines so I don't think it matters too much which you do.